src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -1105,10 +1105,18 @@ void QgsWmsCapabilities::parseLayer( const QDomElement &element, QgsWmsLayerProp
 #endif
 
   layerProperty.orderId     = ++mLayerCount;
-  layerProperty.queryable   = element.attribute( QStringLiteral( "queryable" ) ).toUInt();
+
+  QString queryableAttribute = element.attribute( QStringLiteral( "queryable" ) );
+  layerProperty.queryable = ( queryableAttribute == QLatin1String( "1" ) || queryableAttribute == QLatin1String( "true" ) );
```

Comment:
```suggestion
  layerProperty.queryable = queryableAttribute == QLatin1String( "1" ) || queryableAttribute.compare( QLatin1String( "true" ), Qt::CaseInsensitive ) == 0;
```

... just in case! :wink:

Reply:
:+1: